ttnn/cpp/ttnn/operations/reduction/argmax/device/argmax_program_factory.hpp

```diff
@@ -81,7 +81,7 @@ operation::ProgramWithCallbacks argmax_multi_core(
         C,
         H,
         W,
-        dim.value_or(0),
+        (uint32_t) dim.value_or(0),
```

Comment:
Let’s avoid c-style cast, use static_cast instead. Thank you!

Reply:
OK. I followed the style in the original code.
You mean changing all c-style cast in the file, right?

Reply:
Oh, alright. You did the right thing then. Thank you for the clarification